landable.gemspec

```diff
@@ -49,4 +49,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'valid_attribute'
   gem.add_development_dependency 'pry'
   gem.add_development_dependency 'faker'
+  gem.add_development_dependency 'bunny'
```

Comment:
I do not think we need Bunny as a dependency as it does not have to be enabled by default. Can we just have bunny be included on the parent app and not in landable itself?